### PR TITLE
Body upstream vars

### PR DESCRIPTION
**Pull request by johakoch**

### CODE REVIEW

File: reference/variables.md [hunk 1]
```diff
@@ -228,6 +248,48 @@ paths:
...
+    <headers>
+      <date>Thu, 27 Aug 2020 14:12:33 GMT</date>
+      <content-type>text/html; charset=utf-8</content-type>
+      <connection>keep-alive</connection>
+      <server>gunicorn/19.9.0</server>
+      <access-control-allow-origin>*</access-control-allow-origin>
+      <access-control-allow-credentials>true</access-control-allow-credentials>
+    </headers>
+  </ok>
+  <failure>
```
Comment (filex) [inline reference/variables.md:276]: `ok` and `failure` may be misleading examples. this is the content-id? what about `request1` and `request2`? or `id1` and `id2`?

Comment (johakoch) [reply reference/variables.md:276]: This is the result from the example flow in lines 47ff.

Comment (filex) [reply reference/variables.md]: ah, missed that in the diff view. than "ok" :)

File: reference/variables.md [hunk 2]
```diff
@@ -289,6 +289,12 @@ Example:
 </upstream>
 ```
 
+To check, for example, if the status code of the response with `id="myRequest"` is successful you can use the following XPath expression:
```
Comment (johakoch) [inline reference/variables.md:292]: This looks like there is an `id` attribute with value `myRequest`. In fact, there might be a `content` attribute of the `request` element or an `id` property in the JSON of the `request` element body.

Comment (johakoch) [reply reference/variables.md:292]: -> with the ID `myRequest`

**Pull request merged**